spotify_api.rb

```diff
@@ -0,0 +1,112 @@
+require 'httparty'
+require 'pry'
+require 'json'
+require 'base64'
+require './spotify_api_token'
+
+Spotify_api = "https://api.spotify.com"
+
+class SpotifyApiRequest
+
+  attr_reader :song, :raw_data, :token
+
+  def initialize authorization:, song:, test_file: nil
+    @song = song
+    @authorization = authorization
+    @raw_data = []
+    @test_file = test_file
```

Comment:
As mentioned above, could we end this with `self.parse!`? And make `parse!` a private method?